avc/src/start_detector/start_detector.h

```diff
@@ -0,0 +1,22 @@
+#include <ros/ros.h>
+#include <sensor_msgs/Image.h>
+#include <nodelet/nodelet.h>
+#include <image_transport/image_transport.h>
+#include <cv_bridge/cv_bridge.h>
+#include <std_msgs/Bool.h>
```

Comment:
This isn't a huge deal, but the last two includes here are not needed in this file. They should be moved to the implementation file.